### PR TITLE
Dala 35 architecture diagram

### PR DESCRIPTION
**Pull request by ppuetsch**
# Pull Request Architecture Diagram
This Pull Request adds an Architecture Diagram (draw.io files) for Dataland
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The code has been manually inspected
- [ ] The new feature was observed "in running software"
*No Software was written, only documentation* 
- [x] The DoD was checked for completeness
- [x] The PR actually implements what is described in the JIRA-Issue
- [ ] At least one E2E Test exists testing the new feature
*No Software was written, only documentation* 
- [ ] Documentation is updated as required
*This is documentation :-)* 
- [ ] The new version is deployed to preview using this branch
*No Software was written, only documentation* 
  - [ ] It's verified that this version actually is the one deployed (check actuator/info for branch name and commit id!)
*No Software was written, only documentation* 
  - [ ] The new feature is manually used/tested/observed on preview
*No Software was written, only documentation* 
- [ ] All Classes and Interfaces contain JavaDoc
*No Software was written, only documentation*

### CODE REVIEW

File: documentation/architecture.drawio [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+          <mxGeometry width="50" height="50" relative="1" as="geometry">
+            <mxPoint x="630" y="590" as="sourcePoint" />
+            <mxPoint x="540" y="530" as="targetPoint" />
+          </mxGeometry>
+        </mxCell>
+        <mxCell id="qYgubVpo4Vi_oRc76slj-54" value="The Skyminder Client is closed-Source for Confidentiality Reasons" style="text;html=1;strokeColor=none;fillColor=none;align=center;verticalAlign=middle;whiteSpace=wrap;rounded=0;fontSize=9;" parent="1" vertex="1">
+          <mxGeometry x="395" y="620" width="110" height="30" as="geometry" />
+        </mxCell>
+        <mxCell id="qYgubVpo4Vi_oRc76slj-56" value="&lt;b&gt;OPTIONAL&lt;/b&gt;&lt;br&gt;Data Provider and Data User can opt-in to connect directly to EurDaT or route Data through Dataland" style="text;html=1;strokeColor=none;fillColor=none;align=center;verticalAlign=middle;whiteSpace=wrap;rounded=0;fontSize=9;" parent="1" vertex="1">
```
Comment (SiegfriedSobkowiak) [inline documentation/architecture.drawio:217]: Typo in EurDat should be EuroDat

File: documentation/architecture.drawio [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+        <mxCell id="mH02m_fli5AvtXu53zwJ-27" value="In Scope of Dataland" style="rounded=0;whiteSpace=wrap;html=1;fillColor=#dae8fc;gradientDirection=north;strokeColor=#6c8ebf;" parent="1" vertex="1">
+          <mxGeometry x="960" y="690" width="180" height="30" as="geometry" />
+        </mxCell>
+        <mxCell id="mH02m_fli5AvtXu53zwJ-28" value="In Scope of EuroDaT" style="rounded=0;whiteSpace=wrap;html=1;fillColor=#d5e8d4;gradientDirection=north;strokeColor=#82b366;" parent="1" vertex="1">
+          <mxGeometry x="960" y="730" width="180" height="30" as="geometry" />
+        </mxCell>
+        <mxCell id="mH02m_fli5AvtXu53zwJ-29" value="External Components" style="rounded=0;whiteSpace=wrap;html=1;fillColor=#DEDEDE;gradientDirection=north;strokeColor=#6c8ebf;" parent="1" vertex="1">
+          <mxGeometry x="960" y="770" width="180" height="30" as="geometry" />
+        </mxCell>
+        <mxCell id="qYgubVpo4Vi_oRc76slj-39" value="EurDaT Client&lt;br&gt;Dataspace Connector" style="whiteSpace=wrap;html=1;aspect=fixed;fontSize=13;fillColor=#dae8fc;strokeColor=#6c8ebf;" parent="1" vertex="1">
```
Comment (SiegfriedSobkowiak) [inline documentation/architecture.drawio:307]: Same typo

File: documentation/architecture.drawio [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+        </mxCell>
+        <mxCell id="zA1G0P91OJ-XDcVy-FAF-14" value="" style="edgeStyle=orthogonalEdgeStyle;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;strokeColor=default;exitX=0.25;exitY=1;exitDx=0;exitDy=0;exitPerimeter=0;" edge="1" parent="1" source="zA1G0P91OJ-XDcVy-FAF-19" target="qYgubVpo4Vi_oRc76slj-45">
+          <mxGeometry relative="1" as="geometry">
+            <mxPoint x="340" y="470" as="sourcePoint" />
+          </mxGeometry>
+        </mxCell>
+      </root>
+    </mxGraphModel>
+  </diagram>
+</mxfile>
```
Comment (SiegfriedSobkowiak) [inline documentation/architecture.drawio:335]: Im Bild gibt es auf 2 Netzwerkpfeilen weiße Kreise. Sind diese um zu verdeutlichen das hier Systemgrenzen sind oder was stellen die da? Vielleicht sollte man die in die Legende aufnehmen.

File: documentation/architecture.drawio (hunk 2 above)
Comment (ppuetsch) [reply documentation/architecture.drawio]: fixed

File: documentation/architecture.drawio (hunk 1 above)
Comment (ppuetsch) [reply documentation/architecture.drawio]: fixed

File: documentation/architecture.drawio (hunk 3 above)
Comment (ppuetsch) [reply documentation/architecture.drawio:318]: Habe die Kreise entfernt - ich denke die schaffen keinen Mehrwert.

**Pull request merged**